assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -276,6 +278,7 @@ function BannerNotification( {
 			) }
 			{ description && (
 				<div className="googlesitekit-publisher-win__desc">
+					{ descriptionIcon && <div>{ descriptionIcon }</div> }
```

Comment:
On mobile this doesn't seem needed, but on desktop removing it actually aligns the icon better. The first tab has the div, the second removes it:

![CleanShot 2022-09-07 at 17 56 50](https://user-images.githubusercontent.com/90871/188936512-1d5fe28a-f9b5-4548-a6be-ab28b0fee560.gif)
```suggestion
					{ descriptionIcon }
```

Reply:
So I added that `<div>` to prevent the SVG from resizing and shrinking when you shrink the viewport/window. And not that it would matter, but didn't want an empty `<div>` hanging around there.

Reply:
Fair enough, but it looks like the alignment is off with the `<div>` there—it seems to be pushing it a bit too low.

I'll take a look and see if I can get it working without any resizing issues…